tests/foreman/ui/test_myaccount.py

```diff
@@ -67,38 +67,13 @@ def tearDown(self):
         super(MyAccountTestCase, self).tearDown()
         self.account_user.delete()
 
-    def logged_test_user(self):
+    def session_test_user(self):
```

Comment:
and you don't need to mess with default user and use `with Session(login, pass)` in any place where you need it like we do in other modules